app/routes/api/campaign.js

```diff
@@ -24,7 +24,7 @@ router.route('/campaigns')
                 console.log(err);
             });
     } else {
-        response.status(301).render('unauthorized', {logged_in: false});
```

Comment:
Rendering a server side template for an API response isn't necessary and not really used. Also, 301 is for a redirect. 401 is unauthorized.